crates/cheatcodes/src/evm.rs

```diff
@@ -291,19 +179,20 @@ impl Display for AccountStateDiffs {
         if !&self.state_diff.is_empty() {
             writeln!(f, "- state diff:")?;
             for (slot, slot_changes) in &self.state_diff {
-                match (&slot_changes.slot_info, &slot_changes.decoded) {
-                    (Some(slot_info), Some(decoded)) => {
-                        // Have both slot info and decoded values - only show decoded values
+                match &slot_changes.slot_info {
```

Comment:
we can merge `Some(slot_info) if slot_info.decoded.is_some() => {` with `Some(slot_info) => {`